platform/windows/key_mapping_win.cpp

```diff
@@ -236,11 +236,52 @@ VK_PA1 (0xFD)
 VK_OEM_CLEAR (0xFE)
 */
 
-unsigned int KeyMappingWindows::get_keysym(unsigned int p_code) {
+unsigned int KeyMappingWindows::get_keysym(unsigned int wParam, unsigned int lParam) {
+
+	unsigned int virtual_key = wParam;
+	static bool lshift_down = false;
```

Comment:
Yes I am aware, it was a remnant when I tried to workaround having 2 shift keys pressed at the same time and releasing one of them. Even current stable or master build of Godot has this issue. It is an issue of Windows Messages, low level input handling (e.g, DirectInput) doesn't suffer from this issue. I understand that using DirectInput will require lots of work since it does not read keyboard layouts for different countries/languages, and that is why Godot uses native WM for input which kind of sucks to be honest.

I admit I left this issue open for a long time, I apologize for this, probably I should close this. Despite that it works on Windows. I have no means to test on other platforms like MacOS or Linux. The changes for other platforms should be trivial since most of "constant bindings" are done already which they should work for all platforms,.. I think.